### ISSUE TITLE
Feedstock is broken (Docker issue?)

### ISSUE THREAD

**Issue by wjakob** [conda-forge/pybind11-feedstock#48]
Dear Conda-forge masters,

merging the auto-generated version bump had unintended consequences, the feedstock is now broken with some obscure docker issue that I don't understand. Would you mind taking a look?

Best,
Wenzel

cc @SylvainCorlay, @ax3l

**Comment by tadeu** [comment]
Linux and macOS builds failed in the [latest auto-bump PR](https://github.com/conda-forge/pybind11-feedstock/pull/47):

```
+ test /home/conda/feedstock_root/build_artifacts/pybind11_1585654441832/_test_env_placehold/lib/python3.7/site-packages/pybind11/include == /home/conda/feedstock_root/build_artifacts/pybind11_1585654441832/_test_env_placehold/include
```

it seems the includes are being installed in a directory different than the one expected by the recipe tests. Is this change intentional? If so, maybe a new PR updating these tests in the recipe will fix it.

**Comment by wjakob** [comment]
cc @isuruf

**Comment by wjakob** [comment]
Fixed via #49.
